src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -255,5 +256,13 @@ public TokenCacheNotificationArgs(    // only use this constructor in product co
         /// Cache Details contains the details of L1/ L2 cache for telemetry logging.
         /// </summary>
         public TelemetryData TelemetryData { get; }
+
+        /// <summary>
+        /// Determines if the client application authentication is an FMI node under an RMA.
```

Comment:
```suggestion
        /// Determines whether the client application authentication instance is classified as an FMI (Federated Managed Identity) node under a specified RMA (Resource Managed Application).
```

Reply:
RMA stands for `Resource Management Authority`

Reply:
updated the comment, thanks @bgavrilMS